reV/SAM/econ.py

```diff
@@ -160,57 +160,79 @@ def _get_annual_energy(site, site_df, site_gids, cf_arr, inputs, calc_aey):
         return site_df
 
     @staticmethod
-    def _get_gen_profile(site, site_df, cf_file, cf_year, inputs):
-        """Get the single-site generation time series and add to inputs dict.
+    def _get_cf_profiles(sites, cf_file, cf_year):
```

Comment:
We should really add logic to rex that will check to see if a list can be converted to a slice...

Reply:
Yes but sites may not be consecutive. Going to leave as-is for now.